### ISSUE TITLE
Add HealthCheck URL for Load Balancer monitoring

### ISSUE THREAD

**Issue by miked1313** [thycotic/ss-docs#188]
Document: (https://docs.thycotic.com/ss/10.9.0/networking/secret-server-clustering/index.md)

For load balancer setup/configuration, please add info on how to the load balancer should be setup to monitor the /healthcheck.aspx page.  Please include recommendations such as checking for a 200 OK response code or if checking the page content for "true" should be used.  This was provided to us by Professional Services in a Word document and should be on this documentation page.  Thank you.

**Comment by rossij** [comment]
I agree and I would say this also should be present for RabbitMQ, we didnt get this guidance from Professional services. And thus sent data to an "offline" node... Which caused all sorts of grief.
